playbooks/cloud-provider-openstack-acceptance-test-manila-provisioner/run.yaml

```diff
@@ -0,0 +1,95 @@
+- hosts: all
+  become: yes
+  roles:
+    - config-golang
+    - install-k8s
+    - clone-devstack-gate-to-workspace
+    - create-devstack-local-conf
+    - role: install-devstack
+      enable_services:
+        - 'manila'
+        - 'ceph'
```

Comment:
`enable_services` option should be in role `create-devstack-local-conf`, like this:

```yaml
  roles:
    - config-golang
    - install-k8s
    - clone-devstack-gate-to-workspace
    - create-devstack-local-conf
      enable_services:
        - 'manila'
        - 'ceph'
    - role: install-devstack
```

Reply:
Ah, thanks!